Aztec/Classes/Inserters/Implementations/ImageInserter.swift

```diff
@@ -0,0 +1,56 @@
+import UIKit
+
+class ImageInserter: MediaInserter {
+
+    func attachment(from representation: HTMLRepresentation) -> MediaAttachment? {
+        guard case let .element(element) = representation.kind else {
+            return nil
```

Comment:
This used to return `ImageAttachment(identifier: UUID().uuidString)`, whereas the `VideoFormatter` just threw up it's hands and `assert()`ed. 

I _think_ just returning `nil` here and doing nothing is safe, but I'd appreciate someone double-checking my logic.